src/inausoft.netCLI.Tests/Executor.Tests.cs

```diff
@@ -101,28 +101,28 @@ public void Executor_RunCLI_ShouldRunOnlyProperCommandHandler_WhenMultipleComman
             Assert.AreEqual(0, mockCommand1Handler.LastRunParameters.IntOption);
         }
 
-
-        [ExpectedException(typeof(InvalidCommandException))]
         [TestMethod]
         public void Executor_RunCLI_ShouldThrowInvalidCommmandException_ForNotRegistratedCommand()
```

Comment:
Minor naming issue. Should we name this `...ForNotMappedCommand`? I don't see any `Registrate` or `Register` methods in this library. But I guess we are talking about `Map` method here?

Reply:
I will adjust that as well.